packages/framework/src/Form/Admin/Category/TopCategory/TopCategoriesFormType.php

```diff
@@ -2,9 +2,9 @@
 
```

Comment:
again, look at @PetrHeinz's comment in the analysis - some of the transformers would make sense in `Component\Form` as they are general enough (we can discuss which ones)

Reply:
Not really, the first sentence is the final decision. We decided to move them all to `Form` because it would be really wierd to have few of them somewhere and some of them on different place